src/dsp_tools/commands/create/models/parsed_ontology.py

```diff
@@ -25,7 +28,40 @@ class ParsedClass:
 @dataclass
 class ParsedProperty:
     name: str
-    info: dict[str, Any]
+    labels: dict[str, str]
+    comments: dict[str, str] | None
+    supers: list[str]
+    object: KnoraObjectType | str
+    subject: str | None
+    gui_element: GuiElement
+    list_iri: str | None
```

Comment:
that's not an iri, but a name... ? At parsing time, the IRI doesn't exist yet, because the list hasn't been created yet...?